src/Controls/src/Core/HandlerImpl/VisualElement/VisualElement.Impl.cs

```diff
@@ -306,21 +306,18 @@ static double EnsurePositive(double value)
 
 		void NotifyShadowChanges()
 		{
-			if (Shadow != null)
+			var shadow = Shadow;
+
+			if (shadow is not null)
 			{
-				Shadow.Parent = this;
-				Shadow.PropertyChanged += OnShadowChanged;
+				var proxy = _shadowProxy ??= new();
```

Comment:
Here, like happens in https://github.com/dotnet/maui/commit/61fbb00ebe792c365001afb98b113f01e13cf7d8 we are not setting the Parent. This will cause cases where bindings to the Parent are used to fail etc.

Reply:
I am confused by the line, I don't think we need to add the local ref, we can do:
```suggestion
				 _shadowProxy ??= new();
```
And later use the _shadowProxy.

Reply:
@jsuarezruiz  can you fix this before we land this?